lib/appsignal_phoenix/event_handler.ex

```diff
@@ -36,17 +36,17 @@ defmodule Appsignal.Phoenix.EventHandler do
     end
   end
 
-  def phoenix_endpoint_start(_event, _measurements, _metadata, _config) do
+  def phoenix_router_dispatch_start(_event, _measurements, _metadata, _config) do
     parent = @tracer.current_span()
 
     "http_request"
     |> @tracer.create_span(parent)
-    |> @span.set_attribute("appsignal:category", "call.phoenix_endpoint")
+    |> @span.set_attribute("appsignal:category", "call.phoenix_router_dispatch")
```

Comment:
Every time we add a new the category "group" (the last dot-separated bit), an entry has to be added to [this list](https://github.com/appsignal/appsignal-server/blob/cfa2d58a75dadf895008d02e7af8e45d4ab70c6b/app/frontend/stylesheets/application/_variables.scss#L245-L260) and [that list](https://github.com/appsignal/appsignal-server/blob/cfa2d58a75dadf895008d02e7af8e45d4ab70c6b/app/frontend/javascripts/components/incident/boxes/performance_breakdown/event_groups.jsx#L65-L76).

It might be best to name the categories like `dispatch.phoenix` or `router_dispatch.phoenix` or something like that?

Reply:
I think that’s a good idea. We should look into how the Ruby integration does this and propose better names for these categories. For now, I think we should revert this change to keep it `call.phoenix_endpoint` for now, to get this patch shipped. I’ll update that and will create an issue to revisit the event names.